packages/templates/clients/websocket/javascript/components/MessageExamples.js

```
@@ -3,21 +3,23 @@ import { Text } from '@asyncapi/generator-react-sdk';
+  messages.forEach((message) => {
+    const examples = getMessageExamples(message) || [];
+    examples.forEach((example) => {
+      const payload = example.payload();
+      messageExamples.push(`\n\n**Example:**\n\`\`\`javascript\nclient.${operationId}(${JSON.stringify(payload, null, 2)});\n\`\`\``);
+    });
+  });
   return (
     <Text>
-      {messageExamples}
+      {messageExamples.map(example => (
+        <Text>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)